main/main_test.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"github.com/opentracing/opentracing-go/log"
+	"github.com/stackrox/CvesPusher/utils"
+	"github.com/stretchr/testify/assert"
+	"io/ioutil"
+	"testing"
+)
+
+const (
+	malformedCVEFile = "scratch/malformedCVEs"
+	correctCVEFile   = "scratch/correctCVEs"
+)
+
+func TestEnvVarErrors(t *testing.T) {
+	_, err := utils.IsEnvVarNonEmpty("SOME_LOCAL_PATH")
+	assert.NotNil(t, err)
+
+	_, err = utils.IsEnvVarNonEmpty("SOME_GCLOUD_CONFIG")
+	assert.NotNil(t, err)
+}
+
+func TestNVDParseRightFile(t *testing.T) {
+	dat, err := ioutil.ReadFile(correctCVEFile)
+	if err != nil {
+		log.Error(err)
```

Comment:
Swallowing these errors will cause failing tests to silently pass.

```suggestion
		require.Nil(t, err)
```